ui/client/containers/NussknackerInitializer.tsx

```diff
@@ -136,6 +140,22 @@ class NussknackerInitializer extends React.Component<Props, State> {
 
       const queryHashParams = queryString.parse(this.props.history.location.hash)
       if (settings.implicitGrantEnabled === true && queryHashParams.access_token) {
+        if (settings.jwtAuthServerPublicKey) {
+          let verificationPassed = jwt.verify(queryHashParams.access_token, settings.jwtAuthServerPublicKey)
```

Comment:
I think you should do there `acessTokenVerificationResult` and down `idTokenVerificationResult` each one should be declared as cosnt.